catalog/controller/extension/payment/securesubmit.php

```diff
@@ -461,6 +461,12 @@ private function sessionVar($key = null, $val = null)
          */
         $HPS_KEY = $this->securesubmit_fraud_fail_var;
         //unset($this->session->data[$HPS_KEY]);
+        if (!is_array($this->session->data)){
+            $this->session->data = array();
+        }
+         if (!key_exists($HPS_KEY,$this->session->data) || !is_array($this->session->data[$HPS_KEY])){
+             $this->session->data[$HPS_KEY] = array();
+         }
         if (!key_exists('lastErrorTime',$this->session->data[$HPS_KEY])){
             $this->session->data[$HPS_KEY]['lastErrorTime'] = 0;
         }
```

Comment:
is it possible to replace these three if conditions with this one line code, like 
$this->session->data[$HPS_KEY]['lastErrorTime'] = (!empty($this->session->data[$HPS_KEY]['lastErrorTime'])) ? $this->session->data[$HPS_KEY]['lastErrorTime'] : 0;